src/scss/_header.scss

```diff
@@ -33,4 +33,18 @@
 		height: 1px;
 	}
 
+	@each $breakpoint in ('M', 'L', 'XL') {
+		.o-header__show--#{$breakpoint} {
```

Comment:
How are these classes used exactly? I think some documentation about sticky will be important